app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/masterpasswordguidance/MasterPasswordGuidanceScreen.kt

```diff
@@ -81,126 +78,134 @@ fun MasterPasswordGuidanceScreen(
     }
 }
 
 @Composable
-private fun TryGeneratorCard(
-    onCardClicked: () -> Unit,
+private fun MasterPasswordGuidanceContent(
     modifier: Modifier = Modifier,
+    viewModel: MasterPasswordGuidanceViewModel,
 ) {
-    BitwardenActionCardSmall(
-        actionIcon = rememberVectorPainter(id = R.drawable.ic_generate),
-        actionText = stringResource(
-            R.string.use_the_generator_to_create_a_strong_unique_password,
-        ),
-        callToActionText = stringResource(R.string.try_it_out),
-        onCardClicked = onCardClicked,
-        modifier = modifier
-            .fillMaxWidth(),
-        trailingContent = {
-            Icon(
-                painter = rememberVectorPainter(id = R.drawable.ic_chevron_right),
-                contentDescription = null,
-                tint = BitwardenTheme.colorScheme.icon.primary,
-                modifier = Modifier
-                    .align(Alignment.Center)
-                    .size(16.dp),
-            )
-        },
-    )
+    Column(modifier = modifier) {
+        MasterPasswordGuidanceHeader()
+        MasterPasswordGuidanceContentBlocks()
+        NeedSomeInspirationCard(
+            onActionClicked = remember(viewModel) {
+                {
+                    viewModel.trySendAction(
+                        MasterPasswordGuidanceAction.TryPasswordGeneratorAction,
+                    )
+                }
+            },
+        )
+        Spacer(modifier = Modifier.navigationBarsPadding())
+    }
 }
 
 @Composable
-private fun BulletTextRow(
-    text: String,
-    modifier: Modifier = Modifier,
-) {
-    Row(
-        modifier = modifier
-            .fillMaxWidth()
-            .padding(horizontal = 8.dp),
-    ) {
+private fun MasterPasswordGuidanceHeader(modifier: Modifier = Modifier) {
+    Column(modifier = modifier) {
+        Spacer(modifier = Modifier.height(24.dp))
         Text(
-            text = BULLET_TWO_TAB,
+            text = stringResource(R.string.a_secure_memorable_password),
             textAlign = TextAlign.Center,
-            style = BitwardenTheme.typography.bodyMedium,
+            style = BitwardenTheme.typography.titleMedium,
             color = BitwardenTheme.colorScheme.text.primary,
-            modifier = Modifier.clearAndSetSemantics { },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
         )
+        Spacer(modifier = Modifier.height(8.dp))
         Text(
-            text = text,
+            text = stringResource(
+                R.string.one_of_the_best_ways_to_create_a_secure_and_memorable_password,
+            ),
+            textAlign = TextAlign.Center,
             style = BitwardenTheme.typography.bodyMedium,
             color = BitwardenTheme.colorScheme.text.primary,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+        )
+        Spacer(modifier = Modifier.height(24.dp))
```

Comment:
⛏️ maybe minor thing, curious what others thoughts are with this, maybe we can just remove the wrapping `Column`s and have it so at the call site its like 
```
Column {
  ContentItem1()
  Spacer()
  ContentItem2() 
  etc.
}

Reply:
yeah i like this idea better, ill make the change!